tests/virtctl/guestfs.go

```diff
@@ -119,7 +121,7 @@ func getGuestfsPodName(pvc string) string {
 	return "libguestfs-tools-" + pvc
 }
 
-func execCommandLibguestfsPod(podName, namespace string, args []string) (string, string, error) {
+func execCommandLibguestfsPod(podName, namespace string, args []string) (stdout, stderr string, err error) {
```

Comment:
We should try to avoid naming returned values, it is both confusing and prone to bugs.
Said that, I am unsure if something simple can be done here. I do know why the linter complained.

Reply:
I agree, but in this case the linter asks for it and I don't see a trivial wait to avoid that?

Reply:
I agree, but in this case the linter asks for it and I don't see an easy way to avoid that. Would you prefer a `//nolint:gocritic` instead?

Reply:
Nope, lets keep it like this.